toqito/state_ops/renyi_entropy.py

```diff
@@ -0,0 +1,74 @@
+"""Calculates the renyi entropy for a given density matrix. Valid for special cases of alpha = 0,1 or infty."""
+
+import numpy as np
+
+
+def renyi_entropy(rho: np.ndarray, alpha: float, tolerance: float = 1e-10) -> float:
+    r"""Calculate the Renyi entropy for a quantum density matrix.
+
+    The Rényi entropy of order :math:`\alpha` for a density matrix :math:`\rho` is given by :cite:`quantiki_entropy`
+
+    .. math::
+        S_{\alpha}(\rho) = \frac{1}{1 - \alpha} \log_2 \left( \sum_i \lambda_i^{\alpha} \right),
+
+    where :math:`\lambda_i` are the eigenvalues of :math:`\rho`.
+
+    Special cases :cite:`muller_lennert_renyi_2013`
+        - For :math:`\alpha = 0` (Hartley entropy):
+          .. math:: S_0(\rho) = \log_2 d,
+          where :math:`d` is the rank of :math:`\rho`.
+        - For :math:`\alpha = 1` (Shannon entropy):
+          .. math:: S_1(\rho) = -\sum_i \lambda_i \log_2 \lambda_i.
+        - For :math:`\alpha \to \infty` (Min-entropy):
+          .. math:: S_{\infty}(\rho) = -\log_2 \max_i \lambda_i.
```

Comment:
This is not formatted correctly. 

![image](https://github.com/user-attachments/assets/c925a0b8-6c2d-4844-ae09-6af791804cb1)

https://toqito--1102.org.readthedocs.build/en/1102/autoapi/state_ops/renyi_entropy/index.html

Please check the contributing guide on how you can verify documentation examples etc. locally.